app/views/dashboard/_eeo_data.haml

```diff
@@ -1,6 +1,11 @@
 .date_range
   = form_tag eeo_data_application_records_path, method: :get do
-    List EEO data between
+    List EEO data for
+    .note_item
+      = select_tag :department_id,
+      options_from_collection_for_select(@departments, :id, :name),
+      class: 'dept_select', multiple: true
```

Comment:
Since we can select multiple department IDs, it'd be nice if the various `department_id` variable names here made that clearer - even just `department_ids`, I think, would be preferable.